bumblefoot/Cargo.toml

```
@@ -31,6 +31,7 @@ env_logger = "^0.9.0"
 log = "^0.4.17"
 anyhow = "1"
 console = "^0.15.0"
+exitcode = "1.1.2"
```

A bit more flexible version spec:
```suggestion
exitcode = "^1.1.2"
```